v2/fleet-local/proxy/capcom@.service

```diff
@@ -4,17 +4,17 @@ After=docker.service
 Requires=docker.service
 
 [Service]
-Environment=CP_IMAGE=behance/capcom:latest
-
 User=core
 Restart=on-failure
 RestartSec=20
 TimeoutStartSec=0
-ExecStartPre=/usr/bin/docker pull ${CP_IMAGE}
+Environment="IMAGE=etcdctl get /images/capcom"
+ExecStartPre=/usr/bin/sh -c "source /etc/profile.d/etcdctl.sh && docker pull $($IMAGE)"
```

Comment:
it was always like that. The `sh` runtime does not have the peers endpoint in its environment so the `etcdctl get`s within the run command will fail to connect to etcd